test/parallel/test-help-option.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+require('../common');
+const assert = require('assert');
+const cp = require('child_process');
+const path = require('path');
+
+
+const help = cp.spawnSync(process.execPath, ['-h']);
+
+const man = cp.spawnSync('man', [path.join(process.cwd(), 'doc', 'node.1')]);
+
+if (man.status === 0) {
+  assert.deepStrictEqual(help.stdout, man.stdout);
+} else {
+  const raw = cp.spawnSync(process.execPath, ['--raw-help']);
```

Comment:
That help depends on what features exist. Seems like an unnecessary test burden to maintain.